### PR TITLE
Upgrade snap to stable

### PR DESCRIPTION
**Pull request by psiinon**
OK to upgrade the snap now?

### CODE REVIEW

File: snap/snapcraft.yaml [hunk 1]
```diff
@@ -8,7 +8,7 @@ description: |
   while you are developing and testing your applications.
   Its also a great tool for experienced pentesters to use for manual security testing.
 
-grade: devel
+grade: stable
 confinement: devmode
```
[the remark this answers is not in the document]
Comment (psiinon) [reply snap/snapcraft.yaml]: Ah, that could be 'interesting' as we need write access to ~/.ZAP

**Comment by psiinon** [comment]
For info having problems with the latest snapcraft docker image, will update the PR once thats working :/

**Comment by psiinon** [comment]
Looks like we're blocked on this: https://forum.snapcraft.io/t/permission-denied-when-building-with-snapcore-snapcraft/7186

**Comment by psiinon** [comment]
I've updated the PR but its still failing for me, apparently due to the snapcraft bug

**Comment by psiinon** [comment]
Just tried this again and its now working :D
So ready for review.

**Comment by psiinon** [comment]
Or maybe not .. looks like I was in the wrong branch - sanity checked it and its still failing :((

**Comment by kingthorin** [comment]
Oh no....well the changes still "seem" fine anyway. Based on what I've read etc. (untested).

**Comment by psiinon** [comment]
Looks like this issue is relevant: https://forum.snapcraft.io/t/failed-to-launch-apps-with-classic-confinement/5181/12

**Comment by psiinon** [comment]
For info: https://forum.snapcraft.io/t/zap-snap-needs-manual-review/7575

**Pull request merged**